tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/admin.tsx

```diff
@@ -1,11 +1,34 @@
-import { FeatureColorInput, Feature, FeatureToggle, CheckboxInput } from '../base';
+import { createDropdownInput, Feature, FeatureToggle, CheckboxInput } from '../base';
 
 export const asaycolor: Feature<string> = {
   name: 'Admin chat color',
   category: 'ADMIN',
   subcategory: 'Chat',
   description: 'The color of your messages in Adminsay.',
-  component: FeatureColorInput,
+  component: createDropdownInput(
+    {
+      cfc_red: 'Red',
+      cfc_redorange: 'Redorange',
+      cfc_orange: 'Orange',
+      cfc_yellow: 'Yellow',
+      cfc_lime: 'Lime',
+      cfc_green: 'Green',
+      cfc_cyan: 'Cyan',
+      cfc_bluesky: 'Bluesky',
+      cfc_darkbluesky: 'Dark Bluesky',
+      cfc_navy: 'Navy',
+      cfc_blue: 'Blue',
+      cfc_indigo: 'Indigo',
+      cfc_purple: 'Purple',
+      cfc_violet: 'Violet',
+      cfc_magenta: 'Magenta',
+      cfc_redpurple: 'Redpurple', // pain. manual addition is innevitable
+    },
```

Comment:
OOC chats should have this too, but I have no idea how I can make a module of this. I hate this.